Dockerfile

```diff
@@ -1,4 +1,20 @@
-FROM python:2.7.11-slim
-COPY src/main/python/* /
-ENTRYPOINT ["/bin/bash", "-c", "./runner.sh"]
+FROM python:2.7-slim as build_proto
+RUN  apt-get update \
+  && apt-get install -y wget \
+  && rm -rf /var/lib/apt/lists/*
+
+ADD  grpcio-tools.txt /
+RUN  ["pip","install","-r","grpcio-tools.txt"]
+
+RUN ["wget", "https://raw.githubusercontent.com/projectriff/function-proto/master/function.proto", "-P", "proto"]
+
+# Generate the protobufs
```

Comment:
In the shell/command function invoker, I checked in the generated protobufs on the basis that they would rarely change. That would side-step the issue of fetching from master at build time and make the Dockerfile a lot simpler.

Reply:
sounds good